canonical-web-and-design/vanilla-framework
# Add indeterminate state checkbox

## Done

Added indeterminate state checkbox

Fixes #3293 

## QA

- Open [demo](https://vanilla-framework-3613.demos.haus/docs/examples/patterns/forms/checkbox-indeterminate)
- See that the checkbox shows a dash, and cycles through checked, unchecked and mixed/indeterminate when clicked
- Review updated documentation:
  - [Checkbox docs](https://vanilla-framework-3613.demos.haus/docs/base/forms#checkbox)
  - [Component status](https://vanilla-framework-3613.demos.haus/docs/component-status)

## Screenshots

![Screenshot from 2021-03-04 17-23-17](https://user-images.githubusercontent.com/2376968/110003501-58eeef00-7d0e-11eb-96b0-29f68927beec.png)
